beacon_node/lighthouse_network/src/types/globals.rs

```diff
@@ -72,7 +72,7 @@ impl<E: EthSpec> NetworkGlobals<E> {
             peers: RwLock::new(PeerDB::new(trusted_peers, disable_peer_scoring, log)),
             gossipsub_subscriptions: RwLock::new(HashSet::new()),
             sync_state: RwLock::new(SyncState::Stalled),
-            backfill_state: RwLock::new(BackFillState::NotRequired),
+            backfill_state: RwLock::new(BackFillState::Paused),
```

Comment:
I don't think this state should ever be read, so it doesn't matter what it is

Reply:
Yeah don't see any issues with this